src/app/Nonagent/main.py

```diff
@@ -0,0 +1,100 @@
+from utils import make_request
+
+
+async def version(vicibox_server_ip: str) -> dict:
+    """
+    shows version and build of the API, along with the date/time and timezone
+
+    :param vicibox_server_ip:
+    :return:
+    """
+    url = f'http://{vicibox_server_ip}/vicidial/non_agent_api.php?function=version'
```

Comment:
Suggestion: Maybe we could define this as `BASE_URL` somewhere, or create a util method that will generate this URL for us when we give it a function name. What do you think?

I just thought that we're already defining the base URL here anyway, so why should we need to take `base_url` as parameter for the other functions? Up to you